base/loading.jl

```diff
@@ -151,12 +151,26 @@ function version_slug(uuid::UUID, sha1::SHA1, p::Int=5)
     return slug(crc, p)
 end
 
+struct TOMLCache
+    p::TOML.Parser
+    d::Dict{String, Dict{String, Any}}
+end
+TOMLCache() = TOMLCache(TOML.Parser(), Dict())
```

Comment:
I guess it doesn't matter, but maybe: 
```suggestion
TOMLCache() = TOMLCache(TOML.Parser(), Dict{String,Dict{String,Any}}())
```
?